site/content/en/docs/concepts/admission_check.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Admission Check"
+date: 2023-10-05
+weight: 6
+description: >
+  A mechanism allowing custom internal or external components to control the timing of
+  workloads admission.
+---
+
+## Components
+
+### Admission Check
+
+Admission check is a none namespaced API object used to define details about an `AdmissionCheck` like:
+
+#### controllerName
+
+Is the name with which controller identifies with, not necessarily a K8S Pod or Deployment name. Cannot be empty.
+
+#### retryDelayMinutes
+
+Specifies how long to keep the workload suspended after a failed check (after it transitioned to False). After that the check state goes to "Unknown". The default is 15 min.
+
+#### parameters
+
+Identifies the resource providing additional check parameters.
+
+```yaml
```

Comment:
"An AdmissionCheck object looks like the following:"

Reply:
done